src/business-logic/services/test/rundown-timeline-service.spec.ts

```diff
@@ -12,6 +12,7 @@ import { Blueprint } from '../../../model/value-objects/blueprint'
 import { PartRepository } from '../../../data-access/repositories/interfaces/part-repository'
 import { SegmentRepository } from '../../../data-access/repositories/interfaces/segment-repository'
 import { PieceRepository } from '../../../data-access/repositories/interfaces/piece-repository'
+import {AlreadyActivatedException} from '../../../model/exceptions/already-activated-exception'
```

Comment:
Space before and after the brackets would be nice :)  Such that it looks like this:`{ AlreadyActivatedException }`.